core/src/main/2.0/scala/com/holdenkarau/spark/testing/DatasetSuiteBase.scala

```diff
@@ -41,13 +43,19 @@ trait DatasetSuiteBaseLike extends DataFrameSuiteBaseLike {
     * Compares if two Datasets are equal, Datasets should have the same type.
     * When comparing inexact fields uses tol.
     *
-    * @param tol max acceptable tolerance, should be less than 1.
+    * @param tol          max acceptable decimal tolerance, should be less than 1.
+    * @param tolTimestamp max acceptable timestamp tolerance.
```

Comment:
Same note regarding the description of the tol parameter.

Reply:
Hi @holdenk, not sure if you saw my previous comment, so I pushed the changes so you can see what I mean. Let me know if something does not feel right

Reply:
Hello @holdenk, I'm sending you just a ping so you could take a look over this whenever you have a moment. Thanks!